rasa/cli/data.py

```diff
@@ -336,6 +343,119 @@ def _convert_nlg_data(args: argparse.Namespace) -> None:
         )
 
 
+def _migrate_responses(args: argparse.Namespace) -> None:
+    """Migrate retrieval intent responses to the new 2.0 format.
+
+    It does so modifying the stories and domain files.
+    """
+    _migrate_responses_in_domain(args)
+    rasa.utils.common.run_in_loop(_migrate_responses_in_stories(args))
+    telemetry.track_data_convert(args.format, "responses")
+
+
+def _migrate_responses_in_domain(args: argparse.Namespace):
+    """Migrate retrieval intent responses to the new 2.0 format.
+
+    Before 2.0, retrieval intent responses needed to start
+    with `respond_`. Now, they need to start with `utter_`.
+
+    This function updates the domain file in place.
+
+    Args:
+        args: the CLI arguments
+    """
+    domain_file = Path(args.domain)
+    domain = _get_domain(domain_file)
+
+    domain_dict = domain.cleaned_domain()
+    domain_dict["actions"] = [
+        _normalize_response_name(action) for action in domain_dict["actions"]
+    ]
```

Comment:
quickest way I found to update actions of a domain, not sure if there is another better way?

Reply:
Sounds ok to me 👍